client/my-sites/exporter/export-card/index.jsx

```diff
@@ -77,7 +78,9 @@ class ExportCard extends Component {
 						onClickExport={ this.props.exportSelectedItems }
 					/>
 				</FoldableCard>
-				{ this.props.isExporting && <Interval onTick={ fetchStatus } period={ EVERY_SECOND } /> }
+				{/* Don't fetch status for Jetpack exports, they are not async. */}
```

Comment:
Moved the comment to the beginning of the "return" statement and using `// Comment` now.
This puts the comment far away from the piece of markup it refers to. Is there any way to add comments inside the markup returned from Render() methods?